### DISCUSSION TITLE
How to change int values with menu?

### DISCUSSION THREAD

**Discussion by JimmyRigIt** [forntoh/LcdMenu#47] [Q&A] [answered]
Let me start out by saying, This menu system is AWESOME! So much easier to implement than 500 line sketches. Thanks for all the hard work.

I am building a powder coating oven controller and need to set a number of int variables from 0 to 600..... this would be a massive char list...... surly their must be a better way. 

Next step is saving/loading the values from an SD card so i can save multiple "programs". Will post that here if i get it working.

**Comment by forntoh** [comment, marked as answer]
Solution https://lcdmenu.forntoh.dev/examples/int-and-float-values
